services/app-web/src/components/SubmissionUnlockedBanner/index.ts

```diff
@@ -0,0 +1 @@
+export { SubmissionUnlockedBanner } from './SubmissionUnlockedBanner'
```

Comment:
Knowing that we have more of these alert banners coming up I would  like you to go ahead make the root folder and index.ts be something like `/Alert` or `/Banner` (similar to how we have `/Form`). Then this component  can live inside that folder.